nunit.tests.csharp/TestClass.cs

```diff
@@ -1,20 +1,17 @@
-﻿using NUnit.Framework;
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿// NUnit3 Tests,  see documentation : https://github.com/nunit/docs/wiki/NUnit-Documentation 
```

Comment:
Trailing whitespace, double space after comma, and space before colon, and I wouldn't capitalize `Tests`. I'd prefer to make two sentences instead of the comma.